app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -55,6 +55,24 @@
   }
 }
 
+.gem-c-breadcrumbs[dir="rtl"] {
+  text-align: start;
+
+  .govuk-breadcrumbs__list-item {
+    float: inline-start;
+    margin-left: 0;
+    padding-left: 0;
```

Comment:
Is the `margin-left: 0` and `padding-left: 0` influencing anything? When I disable them in the component example, I can't see any visual changes.

Reply:
Great, thanks!